src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Assets/js/App.tsx

```diff
@@ -123,7 +123,6 @@ function App() {
                 onToggleExplorer={handleToggleExplorer}
             />
             <GraphiQL
-                ref={ref => (_graphiql = ref)}
```

Comment:
Are you sure this needs to be removed?

Reply:
It is not working anymore with it.

No more `ref` property.